lectures/introduction_dynamics/finite_markov.md

```diff
@@ -704,7 +704,7 @@ As seen in {eq}`fin_mc_fr`, we can shift probabilities forward one unit of time
 Some distributions are invariant under this updating process --- for example,
 
 ```{code-cell} julia
-P = [.4 .6; .2 .8];
+P = [0.4 0.6; 0.2 0.8];
 ψ = [0.25, 0.75];
```

Comment:
`psi` in list, but that would be in the other PRs updates as well, so make sure you don't introduce a conflict